istio-cni/templates/_labels.tpl

```diff
@@ -0,0 +1,10 @@
+{{- define "common_labels" }}
```

Comment:
We are trying to avoid these templates as much as possible.

Also we don't want the extra chart/heritage anymore since this will be used by the operator